flytekit/tools/module_loader.py

```diff
@@ -2,10 +2,16 @@
 
 import importlib
 import pkgutil
+import logging as _logging
```

Comment:
I don't see code change in this file... any of these additions needed?

Reply:
reverting